backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -290,43 +290,50 @@ public static void ThrowIfPropertyInSppProject(this Entity.Property property, Cl
         }
 
         /// <summary>
-        /// Get the latest workflow associated to this property, using the workflow sort order.
+        /// Get the latest project associated to this property, using the workflow sort order.
         /// </summary>
         /// <param name="property"></param>
         /// <returns>The workflow code of the latest workflow associated to this property</returns>
-        public static String GetLatestWorkflowCode(this Entity.Property property)
+        public static Entity.Project GetLatestProject(this Entity.Property property)
         {
+            Entity.Project latestProject = null;
             if (property is Entity.Parcel parcel && parcel.Projects.Select(pp => pp.Project).Any())
             {
-                return parcel.Projects.Select(pp => pp.Project).
-                    Aggregate((Entity.Project projectWithLatestWorkflow, Entity.Project current) => current.Workflow.SortOrder > projectWithLatestWorkflow.Workflow.SortOrder ? current : projectWithLatestWorkflow).Workflow.Code;
+                latestProject = parcel.Projects.Select(pp => pp.Project).
```

Comment:
Normally formatting puts the period at the beginning.  Not really an issue though.